buildconfig/Jenkins/Conda/package-conda

```diff
@@ -107,7 +109,7 @@ export LC_ALL=C
 # copy the source to a working folder
 export CONDA_BLD_PATH=$WORKSPACE/conda-bld
 if [[ -d $CONDA_BLD_PATH ]]; then
-    conda index $CONDA_BLD_PATH
+    python -m conda_index $CONDA_BLD_PATH
```

Comment:
I thought this change was already in `main`?

Reply:
The base branch for this is `ornl-next` which has everything from `release-next` rather than `main`.